core/server/data/migrations/versions/3.22/03-add-group-and-flags-to-settings.js

```diff
@@ -0,0 +1,49 @@
+const commands = require('../../../schema').commands;
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up(options) {
+        function addSettingsColumn(column) {
+            return {
```

Comment:
I think it's worth adding a minimal "info" log here saying which column is being added. Symmetricly, could do the same for `removeSettingsColumn` below. The main reason being - it sometimes saves time if the migration fails for whatever reason, and you could see exact step where it broke (used the log a bit when investigating 2.x ->  3.x migration issues)

Reply:
Oh wait, ignore above :laughing: I just had too many other logging turned on and didn't see there's already a log inside of `createColumnMigration`